examples/react/json-based-microapp-renderer-with-css/src/components/Microapps.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2022 Workgrid Software
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react'
+import { useQuery, gql } from '@apollo/client'
+import Microapp from './Microapp'
+
+// GraphQL Query for list of microapps
+const GET_MICROAPPS_QUERY = gql`
+  query GetMicroapps($spaceId: ID!) {
+    me {
+      space(spaceId: $spaceId) {
+        id
+        apps {
+          edges {
+            cursor
+            node {
+              title
+              entrypoint
+            }
+          }
+        }
+      }
+    }
+  }
+`
+
+export default function Microapps() {
+  // GraphQL Query hook to automatically fetch data
+  const { data, loading, error } = useQuery(GET_MICROAPPS_QUERY, {
+    variables: { spaceId: process.env.REACT_APP_SPACE_ID },
+  })
+
+  // Content is not ready, show error/loader
+  if (loading) return <h1 className="microapp-header">Loading microapps...</h1>
+  if (error) return <pre>{error.message}</pre>
+
+  return (
+    <div>
+      {data.me.space.apps.edges?.map((edge) => (
+        <Microapp {...edge} />
+      ))}
+    </div>
+  )
+}
```

Comment:
This example doesn't appear to handle when there are more items to fetch
https://www.apollographql.com/docs/react/pagination/core-api/

Reply:
The api actually doesnt support pagination yet for apps. Will need to add it as a separate issue just to keep this PR smaller. These docs will need to be updated as pagination is added